### PR TITLE
Timber debug log tree planted in both example apps

### LINKED ISSUE #818: Enable logging in the example apps by default
Right now when you use the example apps you have to manually enable the [Timber library](https://github.com/JakeWharton/timber). I think it should be enabled by default and we should add the `Timber.plant()` calls to both publisher and subscriber example apps to enable logs.

### PR DESCRIPTION
**Pull request by davyskiba**
In this PR, I've added the code required for Timber logging to work. We are using calls to Timber across the app for logging, but they do not do anything when no tree is planted.

Resolves #818

### CODE REVIEW

File: subscribing-example-app/src/main/java/com/ably/tracking/example/subscriber/SubscribingExampleApplication.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.ably.tracking.example.subscriber
+
+import androidx.multidex.MultiDexApplication
+import timber.log.Timber
+import timber.log.Timber.DebugTree
+
+class SubscribingExampleApplication : MultiDexApplication() {
```
Comment (KacperKluka) [inline subscribing-example-app/src/main/java/com/ably/tracking/example/subscriber/SubscribingExampleApplication.kt:7]: Do we need to use a `MultiDexApplication`? 🤔

**Review by KacperKluka — COMMENTED** [review]
Looks good, just one question 😉

File: subscribing-example-app/src/main/java/com/ably/tracking/example/subscriber/SubscribingExampleApplication.kt (hunk 1 above)
Comment (KacperKluka) [reply subscribing-example-app/src/main/java/com/ably/tracking/example/subscriber/SubscribingExampleApplication.kt]: This seems to be fixed in https://github.com/ably/ably-asset-tracking-android/pull/821/commits/cc86d0f8239af3e269d97591bf21cd1122b93594

**Review by KacperKluka — APPROVED** [review]
LGTM 👍

**Review by ikbalkaya — APPROVED** [review]
>  they do not do anything when no tree is planted.

I love that 😍

**Pull request merged**